drivers/ieee802154/Kconfig

```diff
@@ -96,4 +88,12 @@ config IEEE802154_UPIPE_MAC7
 
 endif # IEEE802154_UPIPE_RANDOM_MAC
 
+if NET_L2_IEEE802154 || IEEE802154_RAW_MODE
+module = IEEE802154_DRIVER
+module-dep = LOG
+module-str = IEEE 802.15.4 driver
+module-help = Sets log level for IEEE 802.15.4 Device Drivers.
+source "subsys/logging/Kconfig.template.log_config"
```

Comment:
Why do you need to use the generic template here? Note that only network template supports the "module-dep" string.

Reply:
The idea is that 15.4 logging for drivers would be independent from NET_DBG(). `subsys/net/Kconfig.template.log_config.net` mentions specific debug NET_* macros.

Reply:
I will remove `module-dep`, there is already such dependency in template.

Reply:
We probably need, at some later point, to rethink how this NET_RAW_MODE works. It is very confusing option for the casual reader. Better candidate options would probably be CONFIG_NET_L2=n and CONFIG_NET_L3=n which would achieve the same thing but would be more easier to understand.

Reply:
The 15.4 driver does not honor the CONFIG_NET_DEFAULT_LOG_LEVEL setting any more if you use the generic template. This is probably not a big issue but a bit confusing for the end user.